Libraries/LLM/Starcoder2.swift

```diff
@@ -0,0 +1,260 @@
+//
+//  Starcoder2.swift
+//  LLM
+//
+//  Created by John Mai on 2024/3/7.
+//
+
+import Foundation
+import MLX
+import MLXNN
```

Comment:
The CI machinery wasn't set up quite right so didn't check these files, but can you please run:

```
pre-commit run --all-files
```

to apply `swift-format` to all the files?  This one in particular is out of spec.

Thanks!

Reply:
Resolved